buildSrc/src/main/groovy/org/gradle/build/UpdateReleasedVersions.groovy

```diff
@@ -21,24 +21,33 @@ import com.google.common.base.Preconditions
 import groovy.json.JsonBuilder
 import groovy.json.JsonSlurper
 import org.gradle.api.DefaultTask
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
 import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.TaskAction
 import org.gradle.util.GradleVersion
 
+import javax.inject.Inject
+
 class UpdateReleasedVersions extends DefaultTask {
 
     @Internal
     File releasedVersionsFile
 
     @Internal
-    ReleasedVersion currentReleasedVersion
+    Property<ReleasedVersion> currentReleasedVersion
```

Comment:
I guess this property should be final, so we only have a getter.

Reply:
done